rt-dynalib/inc/rt_dynalib.h

```diff
@@ -59,6 +64,14 @@ DYNALIB_FN(15, rt, __errno, int*())
 #if defined(DYNALIB_EXPORT) || (defined(DYNALIB_IMPORT) && !defined(RT_DYNALIB_NO_DEPENDENCY_BREAKING_IMPORTS))
 DYNALIB_FN(16, rt, __assert_func, void(const char*, int, const char*, const char*))
 DYNALIB_FN(17, rt, newlib_impure_ptr_callback, void(void (*)(struct _reent*, size_t, uint32_t, void*), void*))
+// Electron has some of these exported from a different system part through services2 dynalib
+// Photon/P1 do not have enough space in system-part2
+#if HAL_PLATFORM_EXPORT_STDLIB_RT_DYNALIB
```

Comment:
Defining this as a platform feature macro feels a bit weird to me. Perhaps we can define it as a platform-specific hack similarly to `RT_DYNALIB_NO_DEPENDENCY_BREAKING_IMPORTS`?

Reply:
The only problem with that is that we'll have to specify this quirk both when exporting and importing.

Reply:
ah, I see. Please ignore this comment then :+1: